content/en-us/designs/protocol-translation/_index.md

```diff
@@ -0,0 +1,198 @@
+- The gateway runs local protocol translation logic to translate different protocols into a common protocol.
+- If data generated by one of the connected devices has to be pushed to the cloud, the gateway will securely transmit the data in a common format after translation has occurred.
+Since the Protocol Translation design extends the Gateway design, the solution may also allow a device **`B`** to act upon data generated by device **`A`** or vice versa. In this case, the gateway would have protocol translation logic to translate data emitted by device **`A`** into a format for use by the cloud and also protocol translation logic to translate data destined for the cloud into the format usable by device **`B`**.
+ 
+The Protocol Translation design also fills a security purpose:
+The connections inside the local network may be insecure (black) because some of the used protocols have no security features, while the cloud-facing connections (red) are secured by contemporary encryption and authentication mechanisms.
+While encrypting all connections is desirable, in many cases this would prevent using well-established protocols for certain applications that do not support such security mechanisms. Furthermore, for certain devices and applications, replacing devices that do not support contemporary security mechanisms is not economically feasible. Often this means using insecure protocols locally in a solution is an acceptable tradeoff.
+
+## Considerations
+
+When implementing this design, consider the following questions:
+
+#### Does the schema for the translation logic change if the connection is not IP-based?
+In general, the schema does not change.
+In the example [below](#example---industrial-iot-data-extraction), the IP connection was always defined using an IP address.
+For instance, to use a serial connection with Modbus, the corresponding serial hardware path would be used instead of the IP address and a client specifically for Modbus over serial connections needs to be loaded instead of the TCP client, but the design still consistenly needs a device's address.
+
+#### Is there a way to delay/batch/aggregate messages on the gateway?
+Aggregation or batching of data is not a part of this design as it is discussed in the considerations of the Gateway design. 
+The Gateway design contains [considerations]({{ <ref "/designs/gateway/#what-approach-should-be-used-when-storing-messages-for-later-delivery" >}}) about how protocol translation logic can be extended with mechanisms that batch results or compute aggregations over them.
+
+#### How should protocol translation process translated data when the network to the Server is unavailable?
+The Gateway design contains some [considerations](https://iotatlas.net/designs/gateway/#how-should-the-gateway-process-data-when-the-network-to-the-server-is-unavailable) for this scenario. 
```

Comment:
Shouldn't use rendered URLs. Please replace the *considerations* link with:

`[considerations]({{ <ref "/designs/gateway#how-should-the-gateway-process-data-when-the-network-to-the-server-is-unavailable" >}})`